idea_town/frontend/static-src/app/views/base-view.js

```diff
@@ -1,16 +1,36 @@
+import app from 'ampersand-app';
 import AmpersandView from 'ampersand-view';
 import flatten from 'lodash/array/flatten';
 import invoke from 'lodash/collection/invoke';
+import localLinks from 'local-links';
 import mustache from 'mustache';
 // BaseView just abstracts out stuff we seem to use in all the views
 
 export default AmpersandView.extend({
   // override _template with a mustache template
   _template: '',
+
   template(ctx) {
     return mustache.render(this._template, ctx);
   },
 
+  events: {
+    'click a[href]': 'handleLinkClick'
+  },
+
+  // Handles all `<a>` clicks in the app not handled
+  // by another view. This lets us determine if this is
+  // a click that should be handled internally by the app.
```

Comment:
Would you mind explaining the intent here? It seems like we're putting router information into the base view, so that every single view will be doing fallback routing. I'm guessing the intent was more to have a top-level catchall anchor click handler, but I'm not sure why we would need to do that in the first place.